src/main/java/com/rybka/view/ExchangeView.java

```diff
@@ -9,9 +9,11 @@
 public class ExchangeView {
 
     private final Command convertCommand;
+    private final Command historyCommand;
 
     public void showView() {
         convertCommand.execute();
+        historyCommand.execute();
```

Comment:
do we need to show history or convert values on showView? I believe we don't, till user provide appropriate command to execute.

Reply:
fixed